app/experimenter/experiments/migrations/0212_alter_nimbusexperiment_languages_field_for_mobile_client.py

```diff
@@ -0,0 +1,32 @@
+from django.db import migrations
+
+from experimenter.base.models import Language
+from experimenter.experiments.constants import NimbusConstants
+from experimenter.experiments.models import NimbusExperiment as Experiment
+
+
+def update_languages_field_for_mobile_client(apps, schema_editor):
+    NimbusExperiment = apps.get_model("experiments", "NimbusExperiment")
+
+    for experiment in NimbusExperiment.objects.filter(
+        status=NimbusConstants.Status.DRAFT
+    ):
+        if experiment.application != Experiment.Application.DESKTOP:
```

Comment:
This can be done in the queryset with `.exclude(application='firefox-desktop')`

In general anything that can be filtered on in the queryset should be done there rather than in memory so that
1) all the filters are in one place if you need to change them later
2) the database is faster than Python (not that really matters but just worth pointing out)
3) fewer cases to handle in code with logic